docs/metrics/mean-absolute-error.md

```diff
@@ -0,0 +1,66 @@
+# Mean Absolute Error
+
+Mean Absolute Error (MAE) is a popular metric used in assessing regression model performance
+for its overall simplicity and interpretability.
+
+MAE measures the average magnitude of errors in predictions, without considering their direction.
+It represents the mean of the absolute differences between predicted and actual values across a dataset,
+treating each discrepancy equally.
+
+## Implementation Details
+
+MAE is calculated by taking the average of the absolute differences between the predicted values and the actual values.
+This can be mathematically represented as:
+
+$$
+\frac{1}{N} \sum_{i=1}^{N}|x_i-y_i|
+$$
+
+where $x$ is the numerical value from the actual values, and $y$ is the corresponding numerical value from the
+predicted values for a total of $N$ number of predictions.
+
+### Examples
+
+Temperature Estimation:
+
+<div class="grid" markdown>
+| Ground Truth Temperature (&deg;C) | Predicted Temperature (&deg;C) |
+| --- | --- |
+| 25 | 27 |
+| 35 | 30 |
+
+$$
+\begin{align}
+\text{MAE} &= \frac{|25 - 27| + |35 - 30|}{2} \\[1em]
+&= 3.5
+\end{align}
+$$
+</div>
+
+Age Estimation::
+
+<div class="grid" markdown>
+| Ground Truth Age (Years) | Predicted Age (Years) |
+| --- | --- |
+| 60 | 70 |
+| 40 | 20 |
+
+$$
+\begin{align}
+\text{MAE} &= \frac{|60 - 70| + |40 - 30|}{2} \\[1em]
```

Comment:
Should the predicted age on the second datapoint be 20 or 30?

Reply:
Apologies - yeah it should be 20.